roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -212,3 +212,47 @@ pbindings(){
     cd $CURRENT_DIR
 }
 _pbindings_help="Create and install bindings. Example usage: pbindings pulpcore python"
+
+pminio(){
+    if [ $1 != 'stop' ]
+    then
+        sudo bash -c "sed -i '/AWS_ACCESS_KEY_ID/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_SECRET_ACCESS_KEY/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_STORAGE_BUCKET_NAME/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_DEFAULT_ACL/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/S3_USE_SIGV4/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_S3_SIGNATURE_VERSION/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_S3_ADDRESSING_STYLE/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_S3_REGION_NAME/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/DEFAULT_FILE_STORAGE/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/MEDIA_ROOT/d' /etc/pulp/settings.py"
+        sudo bash -c "sed -i '/AWS_S3_ENDPOINT_URL/d' /etc/pulp/settings.py"
+    else
+        export MINIO_ACCESS_KEY=AKIAIT2Z5TDYPX3ARJBA
+        export MINIO_SECRET_KEY=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
+        podman run -d -p 9000:9000 -e MINIO_ACCESS_KEY=$MINIO_ACCESS_KEY -e MINIO_SECRET_KEY=$MINIO_SECRET_KEY minio/minio server /data
+        wget https://dl.min.io/client/mc/release/linux-amd64/mc
+        chmod +x mc
+        sudo mv mc /usr/local/bin
+        while ! nc -z localhost 9000; do echo 'Wait minio to startup...' && sleep 0.1; done;
+        mc config host add s3 http://localhost:9000 AKIAIT2Z5TDYPX3ARJBA fqRvjWaPU5o0fCqQuUWbj9Fainj2pVZtBCiDiieS --api S3v4
+        mc config host rm local
+        mc mb s3/pulp3 --region eu-central-1
+        workon pulp
+        echo "Declaring S3 environment variables..."
+        sudo bash -c "echo AWS_ACCESS_KEY_ID = \'AKIAIT2Z5TDYPX3ARJBA\' >> /etc/pulp/settings.py"
+        sudo bash -c "echo AWS_SECRET_ACCESS_KEY = \'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\' >> /etc/pulp/settings.py"
```

Comment:
Any chance you can reuse the variables you declare above ($MINIO_ACCESS_KEY and $MINIO_SECRET_KEY) here?